ote_cli/ote_cli/datasets/anomaly/__init__,py

```diff
@@ -0,0 +1,15 @@
+"""DataLoader for anomaly tasks"
```

Comment:
the file name has a comma instead of a point

Reply:
Whoops 🙃